python/dgl/dataloading/pytorch/__init__.py

```diff
@@ -347,10 +369,21 @@ def __init__(self, g, nids, block_sampler, device='cpu', **kwargs):
                         dataloader_kwargs['shuffle'] = False
                         dataloader_kwargs['drop_last'] = False
 
+            self.use_ddp = use_ddp
+            if use_ddp:
+                self.dist_sampler = DistributedSampler(
+                    dataset,
+                    shuffle=dataloader_kwargs['shuffle'],
+                    drop_last=dataloader_kwargs['drop_last'])
+                dataloader_kwargs['shuffle'] = False
```

Comment:
What's the point of resetting kwargs here?

Reply:
PyTorch spec requires that `shuffle` must not be specified (i.e. False) when `sampler` is specified.